src/main/java/duke/ui/MainWindow.java

```diff
@@ -0,0 +1,59 @@
+package duke.ui;
+
+import duke.Duke;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
```

Comment:
Astute idea to separate out MainWindow as a UI component. Makes for a clean SOC between Duke and its UI!